### ISSUE TITLE
Custom Reference Genome without bsgenome file

### ISSUE THREAD

**Issue by Kaivalyamolugu** [GreenleafLab/ArchR#566]
Hello, 

I am working with snATAC-seq data for a GFP reporter human cell line (https://www.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=AICS-0061-036&PgId=166) that does not have bsgenome file available. I am trying to see the chromatin accessibility around the GFP reporter. Would you have recommendations of how to create a custom reference genome for this?

Thanks, 
Kaivalya

**Comment by rcorces** [comment]
Neither of us have ever done this so unfortunately we dont have much advice. The process of creating your own BSgenome seems relatively involved. Might be a question better addressed to more general forums. In case this helps:
https://www.bioconductor.org/packages/devel/bioc/vignettes/BSgenome/inst/doc/BSgenomeForge.pdf
